octangroup/git-cafe-exercise
# Bugfix/coantact page issue

# Exercise 2 - Bug Fix

## Description

This pull request addresses a bug in the `git-cafe-exercise` repository by fixing the title of the `index-4.html` file to "**Contact**".

#### What does this PR do?

This PR fixes the title of the `index-4.html` file in the `git-cafe-exercise` repository, correcting it to "**Contact**".

#### Description of Task Completed

- Created a new bug fix branch named `Bugfix/coantact page issue`.
- Modified the `index-4.html` file to change the title to "**Contact**".

#### How should this be manually tested?

To manually test this PR:
1. Clone the repository to your local machine.
2. Checkout the `Bugfix/coantact page issue` branch.
3. Open the `index-4.html` file in a web browser.
4. Verify that the title of the page is displayed as "**Contact**".
